src/common/interpolation.c

```diff
@@ -1693,10 +1635,9 @@ static void dt_interpolation_resample_sse(const struct dt_interpolation *itor, f
 
   _mm_sfence();
 
-#if DEBUG_RESAMPLING_TIMING
-  ts_resampling = getts() - ts_resampling;
-  fprintf(stderr, "resampling %p plan:%" PRId64 "us resampling:%" PRId64 "us\n", in, ts_plan, ts_resampling);
-#endif
+  dt_show_times_f(&start, "[dt_interpolation_resample_sse]",
```

Comment:
This will now report the sum of planning and resampling times since the start time didn't get updated as it used to with `int64_t ts_resampling = getts()` at (original) line 1615.

Reply:
You're right. Time used planning is generally negligible, but I realize that I only tested it on images with preview resolution reduced to 1/4, and was seeing results like:

```
    13.4708 [dt_interpolation_resample_plain] took 0.000 secs (0.003 CPU) planning 4383x2920 -> 1141x760
    13.5594 [dt_interpolation_resample_plain] took 0.089 secs (1.258 CPU) resampling 4383x2920 -> 1141x760 via lanczos3
```

Here's some other more typical results (all with code patched to show planning time):

```
    10.9712 [dt_interpolation_resample_cl] took 0.002 secs (0.008 CPU) planning 4383x2920 -> 1141x760
    10.9749 [dt_interpolation_resample_cl] took 0.001 secs (0.004 CPU) resampling 4383x2920 -> 1141x760 via lanczos3
```

```
    99.8164 [dt_interpolation_resample_sse] took 0.001 secs (0.007 CPU) planning 4383x2920  -> 1141x760
    99.8944 [dt_interpolation_resample_sse] took 0.078 secs (0.928 CPU) resampling 4383x2920 -> 1141x760 via lanczos3
```

```
     4.4151 [dt_interpolation_resample_plain] took 0.002 secs (0.004 CPU) planning 4383x2920 -> 1141x760
     4.5092 [dt_interpolation_resample_plain] took 0.094 secs (1.367 CPU) resampling 4383x2920 -> 1141x760 via lanczos3
```

The planning time is still negligible, but it's not 0.000, and in the case of OpenCL path is actually the majority of the time!

Clearly it was worth reporting this when developing the original code. The original logging showed planning and resampling time on one line, which was nice. I didn't want too much noise in the logs, and wanted to keep the code succinct, hence `dt_show_times_f()` and one line.

It's a minor patch to start reporting it. @ralfbrown or @TurboGit: Do you have strong feelings about this? I'm happy to just keep things proper and add the planning time logging back in.

Reply:
I went ahead and made #13541 to bring back the former behavior...